### PR TITLE
Feat: Shortest

### PR DESCRIPTION
**Pull request by m2rads**
# AI-Powered E2E Testing Framework

Addresses: #69

This PR introduces a complete end-to-end testing framework powered by Claude AI. The framework enables writing tests in natural language and executes them through AI-driven browser automation.

## Key Features
- Natural language test definitions using a fluent API
- AI-powered test execution using Claude computer use
- Automated browser interaction with precise coordinate handling
- Detailed test reporting with pass/fail status

## Technical Highlights
- Direct integration with Anthropic's Claude API
- Playwright-based browser automation
- Coordinate scaling system for reliable element targeting
- Visual mouse tracking for debugging
- Modular architecture with clear separation of concerns

### CODE REVIEW

**Comment by m2rads** [comment]
Hi @slavingia! Please do not merge this PR just yet. 
I noticed an issue with Vercel deployment. Let me resolve that first. 

Please clone this branch locally if you'd like to test the lib.

**Comment by m2rads** [comment]
Documentation is in docs/README.md

**Comment by m2rads** [comment]
Hi @slavingia looking at my Vercel looks like the deployment was done successfully. Can you check on your side to see if there are any issues?

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+# Shortest
```
Comment (slavingia) [inline docs/README.md:1]: Why not just update the README?

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+[![npm version](https://badge.fury.io/js/shortest.svg)](https://badge.fury.io/js/shortest)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
```
Comment (slavingia) [inline docs/README.md:5]: Not a fan of anything that isn't essential, images that need to be loaded, etc. license is in another file already

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+> 🤖 An AI-powered E2E testing framework that uses natural language to write and execute tests.
+
+## ✨ Features
+
+- 🤖 AI-powered test execution
```
Comment (slavingia) [inline docs/README.md:11]: Seems AI written and not useful. Better to be on the homepage than README if we think it's needed

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+- 🖱️ Visual mouse tracking
+- 📊 Detailed test reports
+
+## 📦 Installation
+
+Choose your preferred package manager:
+
+```bash
+# pnpm
+pnpm add -D shortest
```
Comment (slavingia) [inline docs/README.md:24]: Prefer just doing one, npm to start

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+define('Validate OAuth login', () => {
+  new UITestBuilder('/')
+    .test('Login with Github')
+    .given('click Github login button')
+    .expect('should complete OAuth flow')
+    .expect('should redirect to /dashboard');
+});
+```
+
+## ⚙️ Configuration Options
```
Comment (slavingia) [inline docs/README.md:154]: IMO should be in the example config itself versus here (and all below this can go too)

File: package.json [hunk 6]
```diff
@@ -79,6 +79,7 @@
     "js-yaml": "^4.1.0",
     "jsdom": "^25.0.1",
     "smee-client": "^2.0.3",
-    "vitest": "^2.1.1"
+    "vitest": "^2.1.1",
+    "shortest": "workspace:*"
```
Comment (slavingia) [inline package.json:83]: What's this do?

File: packages/shortest/package.json [hunk 7]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "shortest",
+  "version": "0.0.1",
+  "description": "AI-powered testing framework",
```
Comment (slavingia) [inline packages/shortest/package.json:4]:
```suggestion
  "description": "AI-powered natural language end-to-end testing framework",
```

**Review by slavingia — APPROVED** [review]
A few small comments here and there. I'll also ask some Antiwork senior engineers to review.

File: package.json (hunk 6 above)
Comment (m2rads) [reply package.json:83]: So I setup pnpm-workspace.yaml that allows for a better monorepo development of the package. This way we can build and install the package locally as if we have installed it from npm. I think it's good for testing the package during development.

File: docs/README.md (hunk 1 above)
Comment (m2rads) [reply docs/README.md]: Done!

**Comment by m2rads** [comment]
Should I add unit tests for this package?

File: README.md [hunk 8]
```diff
@@ -123,8 +123,51 @@ Open [http://localhost:3000](http://localhost:3000) in your browser to see the a
...
+To run unit tests:
 
 ```bash
 pnpm vitest
 ```
+
+
+## Setup Guide for Running and Building Shortest Package Locally
+   
+   When running `pnpm install` in the root, the `shortest` package will built and added in the `node_modules` folder. However, in order to run the tests locally using `pnpm test` or `shortest cli`, you need to do the following:
```
Comment (slavingia) [inline README.md:135]: A bit weird spacing here

Comment (m2rads) [reply README.md]: Just saw this. Fixed

**Comment by slavingia** [comment]
Merged but if you wish to add unit tests as a new PR you're welcome to do so. I like the idea of trying to rely 100% on shortest tests.

**Pull request merged**